### DISCUSSION TITLE
Max height isometric commit chart

### DISCUSSION THREAD

**Discussion by Raybie** [lowlighter/metrics#1554] [Q&A]
Is there any way to set a maximum chart height so a single day where let's say 40 commits were made doesn't make all the other days on the charts look like small ants?

**Comment by lowlighter** [comment]
Hi 👋 !

I'll add an option for it in the v4 refactor of the isocalendar, thanks for the suggestion 🙂

**Reply by Raybie** [reply]
Ohh, awesome! Thank you :D
